src/Admin.ts

```diff
@@ -81,14 +82,16 @@ export default class Admin {
         }
 
         let adminChannel = guild.channels.cache.find(c => c.name === this.sharedSettings.server.guruChannel);
+        let adminLogChannel;
         if (!adminChannel) {
             if (this.sharedSettings.botty.isProduction) {
                 console.error(`Admin: Unable to find moderators channel!`);
                 return;
             }
             else {
-                adminChannel = await guild.channels.create({name: this.sharedSettings.server.guruChannel, type: Discord.ChannelType.GuildText});
+                adminChannel = await guild.channels.create({name: this.sharedSettings.server.guruLogChannel, type: Discord.ChannelType.GuildText});
             }
+            adminLogChannel = guild.channels.cache.find(c => c.name === this.sharedSettings.server.guruLogChannel) || adminChannel;
```

Comment:
We only initialise the log channel if we can't find the admin channel?

Reply:
Fixed